examples/notebooks/get-started/try-apache-beam-yaml.ipynb

```diff
@@ -0,0 +1,424 @@
+{
```

Comment:
It would be good to get someone from the Tech Writer team to review too. You can check w/ rszper@.

Reply:
Good idea. I'll reach out to rszper@.